runtime/oti/j9javaaccessflags.h

```diff
@@ -30,112 +30,132 @@
+#define J9AccClassHasNonStaticNonAbstractMethods 0x8000000
 #define J9AccClassReferenceWeak 0x10000000
-#define J9AccClassRomToRamMask 0xF3000000
-#define J9AccClassUnsafe 0x40000
-#define J9AccClassUseBisectionSearch 0x2000
-#define J9AccEmptyMethod 0x4000
-#define J9AccEnum 0x4000
-#define J9AccFinal 0x10
-#define J9AccForwarderMethod 0x2000
+#define J9AccClassReferenceSoft 0x20000000
+#define J9AccClassReferencePhantom 0x30000000
+#define J9AccClassFinalizeNeeded 0x40000000
+#define J9AccClassCloneable 0x80000000
+
+/* ROMMethod modifiers (includes those that aren't also applicable to a class)
+ * See map in ROMClassWriter::writeMethods for
+ * available slots.
+ */
+#define J9AccSynchronized 0x20
+#define J9AccBridge 0x40
+#define J9AccVarArgs 0x80
+#define J9AccNative 0x100
 #define J9AccGetterMethod 0x200
-#define J9AccInterface 0x200
-#define J9AccMandated 0x8000
-#define J9AccMethodCallerSensitive 0x100000
-#define J9AccMethodAllowFinalFieldWrites 0x1000000
+#define J9AccStrict 0x800
+#define J9AccForwarderMethod 0x2000
+#define J9AccEmptyMethod 0x4000
+#define J9AccMethodVTable 0x10000
+#define J9AccMethodHasExceptionInfo 0x20000
+#define J9AccMethodHasDebugInfo 0x40000
 #define J9AccMethodFrameIteratorSkip 0x80000
+#define J9AccMethodCallerSensitive 0x100000
 #define J9AccMethodHasBackwardBranches 0x200000
-#define J9AccMethodHasDebugInfo 0x40000
-#define J9AccMethodHasDefaultAnnotation 0x80000000
-#define J9AccMethodHasExceptionInfo 0x20000
-#define J9AccMethodHasExtendedModifiers 0x4000000
+#define J9AccMethodObjectConstructor 0x400000
+#define J9AccMethodHasMethodParameters 0x800000
+#define J9AccMethodAllowFinalFieldWrites 0x1000000
 #define J9AccMethodHasGenericSignature 0x2000000
-#define J9AccMethodHasMethodAnnotations 0x20000000
+#define J9AccMethodHasExtendedModifiers 0x4000000
 #define J9AccMethodHasMethodHandleInvokes 0x8000000
-#define J9AccMethodHasMethodParameters 0x800000
-#define J9AccMethodHasParameterAnnotations 0x40000000
 #define J9AccMethodHasStackMap 0x10000000
-#define J9AccMethodHasTypeAnnotations 0x4000000
-#define J9AccMethodObjectConstructor 0x400000
+#define J9AccMethodHasMethodAnnotations 0x20000000
+#define J9AccMethodHasParameterAnnotations 0x40000000
+#define J9AccMethodHasDefaultAnnotation 0x80000000
+/* Masks and helpers */
+#define J9AccMethodReturnMask 0x1C0000
+#define J9AccMethodReturnShift 0x12
+/* Used in HookedByTheJit.cpp */
 #define J9AccMethodReturn0 0x0
 #define J9AccMethodReturn1 0x40000
 #define J9AccMethodReturn2 0x80000
 #define J9AccMethodReturnA 0x140000
 #define J9AccMethodReturnD 0x100000
 #define J9AccMethodReturnF 0xC0000
-#define J9AccMethodReturnMask 0x1C0000
-#define J9AccMethodReturnShift 0x12
-#define J9AccMethodVTable 0x10000
-#define J9AccNative 0x100
-#define J9AccPrivate 0x2
-#define J9AccProtected 0x4
-#define J9AccPublic 0x1
-#define J9AccStatic 0x8
-#define J9AccStrict 0x800
-#define J9AccSuper 0x20
-#define J9AccSynchronized 0x20
-#define J9AccSynthetic 0x1000
```

Comment:
If `J9AccSynthetic` is removed from` ROMClass->extraModifiers`, you may want to modify this comment as well:

https://github.com/eclipse-openj9/openj9/blob/f7305e7f338a61150e71235eea611d88f946b887/runtime/bcutil/ROMClassBuilder.cpp#L1183

Reply:
Please correct the indentation of line 1176 (there should be one more space before `+`).

Reply:
@theresa-m ?

Reply:
Hey sorry I missed this, it should be fixed now.